source/_integrations/mealie.markdown

```diff
@@ -14,8 +14,21 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
+
 The Mealie integration will fetch data from your [Mealie instance](https://mealie.io/).
 
+## Getting an API token
+
+You create your API token on your Mealie installation:
+
+1. Sign in to Mealie
+2. Go to your user (profile)
+3. Go to Manage Your API Tokens (`/user/profile/api-tokens`)
+4. Enter a meaningful token name, such as 'Home Assistant'
+5. Click _Generate_
```

Comment:
```suggestion
5. Select **Generate**.
```
We usually use **select** instead of [click](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)